Assets/SuperMultiplayerShooter/Scripts/SampleMainMenu.cs

```diff
@@ -163,123 +302,198 @@ void Update()
         }
 
         /// <summary>
-        /// Called whenever the scene or game ends. Unsubscribe from all channels and channel groups to trigger events immediately.
+        /// Called whenever the scene or game ends. Unsubscribe event listeners.
         /// </summary>
         void OnDestroy()
         {
-            _pubnub.UnsubscribeAll()
```

Comment:
Is it possible to unsubscribe from all channels as part of the shutdown process?  I tried this with some of my channels in OnDestroy() with the old SDK (as well as in the Unity ApplicationWantsToQuit() callback) but I couldn't do it as the old SDK didn't support await.

Reply:
Implemented null reference check in this [commit](https://github.com/PubNubDevelopers/unity-multiplayer-shooter/pull/3/commits/657d070c41735530e196769610186fb7f0bff9ff) to ensure no null reference errors occur when trying to unsubscribe from all channels.